tests/Application/src/Kernel.php

```diff
@@ -22,12 +25,27 @@ final class Kernel extends BaseKernel
 
     public function registerBundles(): iterable
     {
-        $bundlesPath = $this->getProjectDir() . '/tests/Application/config/bundles.php';
+        $bundlesPath = $this->getProjectDir() . '/config/bundles.php';
         $contents = require $bundlesPath;
         foreach ($contents as $class => $envs) {
             if ($envs[$this->environment] ?? $envs['all'] ?? false) {
                 yield new $class();
             }
         }
     }
+
+    private function configureContainer(ContainerConfigurator $container, LoaderInterface $loader, ContainerBuilder $builder): void
```

Comment:
I'm not sure if this configuration will resolve the initial problem described in the issue, such that an application not using Behat throws an exception when run in a test environment 🤔

Reply:
I forgot to remove this method. It's copied from the `MicroKernelTrait` trait so it doesn't matter. To fix the issue I moved services declarations from `/config` to `/tests/Application`.